QUARKUS-2991.md

```diff
@@ -0,0 +1,128 @@
+# QUARKUS-2991 Support for MicroProfile 6
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-2991
+
+Quarkus documentation:
+- https://quarkus.io/version/main/guides/opentelemetry
+- https://quarkus.io/version/main/guides/openapi-swaggerui
+- https://quarkus.io/version/main/guides/security-jwt
+- https://quarkus.io/version/main/guides/smallrye-metrics
+
```

Comment:
Add pointers to MP stuff

 - https://microprofile.io/2023/01/10/microprofile-6-0-release/#
 - https://microprofile.io/2021/12/07/microprofile-5-0-release/#